app-config-default-extensions/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@app-config/default-extensions",
```

Comment:
Might be able to get rid of default-extensions altogether, but that might bite us if new extensions were added